src/components/Form.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+
+
+function Form({setStatus, setInputText, todos, setTodos, inputText}) {
+    
+    const inputTextHandler = (e) => {
+        setInputText(e.target.value);
+     };
+
+     const submitTodoHandler = (e) => {
+         e.preventDefault();
+         setTodos([
+             ...todos, {text: inputText, completed: false, id: Math.random() }
```

Comment:
Check first if the generated `Math.random()` has a duplicate value in the existing todo cause it might cause duplicate id bug.

Reply:
Also preferred to specify objects like this for readability purposes.

```js
{
  ...todos,
  {
    id: generatedId,
    text: inputText,
    completed: false,
  }
}
```

Reply:
used Deno (web module) Import
import ShortUniqueId from 'https://cdn.jsdelivr.net/npm/short-unique-id@latest/short_uuid/mod.ts';
instead of Math.random(); for unique id's and no duplication

Reply:
revert changes due to Error